packages/models-library/src/models_library/api_schemas_api_server/api_keys.py

```diff
@@ -6,7 +6,10 @@ class ApiKey(BaseModel):
     api_secret: SecretStr
 
 
-class ApiKeyInDB(ApiKey):
+class ApiKeyInDB(BaseModel):
+    api_key: str
+    api_secret: str
+
```

Comment:
Did you run the api server unit tests? I am worried this will break some of the fixtures there.